string command
float64 goalX
float64 goalY
float64 distance
string frame
float64 goalTheta
float64 angle
---
bool done
